src/sql/workbench/services/notebook/common/notebookServiceImpl.ts

```diff
@@ -424,8 +424,8 @@ export class NotebookService extends Disposable implements INotebookService {
 	}
 
 	private waitOnProviderAvailability(providerDescriptor: ProviderDescriptor, timeout?: number): Promise<INotebookProvider> {
-		// Wait up to 10 seconds for the provider to be registered
-		timeout = timeout || 10000;
+		// Wait up to 30 seconds for the provider to be registered
+		timeout = timeout || 30000;
```

Comment:
Should we use a timeout this high? Seems like a long time to wait for a failure, especially since all this stuff happens locally.

Reply:
Disregard this, saw the description write-up.